src/iosrv/srv.c

```diff
@@ -253,6 +332,7 @@ dss_sched_run(ABT_sched sched)
 
 	while (1) {
 		/* Execute one work unit from the scheduler's pool */
+		ds_check_expire_list(false);
```

Comment:
The ULTs count should be get by ABT_pool_get_size() but not ABT_pool_get_total_size().

Reply:
right.